soc/nxp/imx/imx8/Kconfig.soc

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMX_8
+	bool
+	select SOC_FAMILY_NXP_IMX
+
+config SOC_SERIES
+	string
+	default "imx8" if SOC_SERIES_IMX_8
```

Comment:
Align on value and have the same for both e.g `imx8` and `SOC_SERIES_IMX8`

Reply:
I've updated all to: SOC_SERIES_IMX8, SOC_SERIES_IMX8M, SOC_SERIES_IMX8X, SOC_SERIES_IMX8ULP